x-pack/plugin/inference/qa/inference-service-tests/src/javaRestTest/java/org/elasticsearch/xpack/inference/InferenceCrudIT.java

```diff
@@ -142,6 +144,34 @@ public void testApisWithoutTaskType() throws IOException {
         deleteModel(modelId);
     }
 
+    @SuppressWarnings("unchecked")
+    public void testPerformInferenceWithChunkingForSparseEmbeddingEndpoint() throws IOException {
+        String modelId = "chunking_sparse_embedding";
+        putModel(modelId, mockSparseServiceModelConfig(), TaskType.SPARSE_EMBEDDING);
+        var input = List.of("hello world", "this is the second document");
+        var bodyParameters = new HashMap<String, Object>();
+        bodyParameters.put("chunk", true);
+
+        var inference = infer(modelId, TaskType.SPARSE_EMBEDDING, input, bodyParameters, null, Map.of());
```

Comment:
Please add coverage for calling inference without the tasktype in the URL - `POST _inference/inference_id`